test/sqatt/src/Sqatt.hs

```diff
@@ -317,6 +317,7 @@ runTxsWithExample mLogDir ex delay = Concurrently $ do
     txsUIProc mUiLogDir imf port =
       Concurrently $ do
         eRes <- try $ Turtle.fold txsUIShell findExpectedMsg
+        sleep 5.0       -- take time to write results to log files (in case of error)
```

Comment:
We should remove this sleep in the future. I've created https://github.com/TorXakis/TorXakis/issues/601